lib/App/DuckPAN/Query.pm

```diff
@@ -2,95 +2,125 @@ package App::DuckPAN::Query;
 # ABSTRACT: Main application/loop for duckpan query
 
 use Moo;
-
-my $query;
-my $history_path;
-
+use DDG;
+use DDG::Request;
```

Comment:
Ah I get it. `App::DuckPAN::DDG` gets loaded by `App::DuckPAN` at https://github.com/duckduckgo/p5-app-duckpan/blob/f6dbd313e7efec927d109bd65b1f8b30c12dd6db/lib/App/DuckPAN.pm#L262

It's `require`d and `import`ed only once we need it to run Query.pm

@zachthompson you're much more experienced with Perl than me -- do you have any ideas about how to fix this somewhat convoluted situation? Should we just revert back to using `require` and `import` inside `run`?